docs/eigenda/operator-guides/eigenda-avs-installation-registration-and-upgrade/holesky.md

```diff
@@ -3,4 +3,188 @@ sidebar_position: 2
+
+In order for users to retrieve data from your node, you will need to open access to retrieval ports.
+
+Ensure the port specified as `NODE_RETRIEVAL_PORT` in the [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L17) has open access to the public internet.
+
+Dispersal Setup:
+
+In order to limit traffic from the EigenLabs hosted Disperser, please restrict your node's ingress traffic to be allowed by the the list provided below and port number set as `NODE_DISPERSAL_PORT` in the [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L14) in the below setup.
+
+- `3.221.120.68/32`
+- `52.2.226.152/32`
+- `18.214.113.214/32`
+
+#### Step 4: Opt-in into EigenDA
+
+In order to opt-in into EigenDA as an Operator, you must meet the following delegated TVL requirements:
+
+- Have a minimum of 32 ETH delegated.
+- Have more than 1.1x current lowest-stake Operator in the active Operator set. Please see [EigenDA Churn Approver](https://docs.eigenlayer.xyz/operator-guides/avs-installation-and-registration/eigenda-operator-guide#eigenda-churn-approver) for more detail.
+- The operator to churn out has less than 10.01% of the total stake
+
+Execute the following command to opt-in to EigenDA AVS: 
+This command also downloads the latest SRS points if they don't exist on the node. The file is approximately 8GB in size and the opt-in process can some time to complete depending on the network bandwidth.
+
+```
+./run.sh opt-in
+```
+
+Note: the script will use the `NODE_HOSTNAME` from [.env](https://github.com/Layr-Labs/eigenda-operator-setup/blob/2872d76b5e0b127400eb7e6dd16da362c7c142ba/.env.example#L63) as your current IP.
+
+If your operator fails to opt-in to EigenDA or is ejected by the Churn Approver then you may run the opt-in command again after the rate limiting threshold has passed. The current rate limiting threshold is 5 minutes.
+
+If you receive the error “error: failed to request churn approval .. Rate Limit Exceeded” you may retry after the threshold has passed. If you receive the error “insufficient funds”, you may increase your Operator’s delegated TVL to the required minimum and retry after the threshold has passed.
+
+#### Step 5: Run EigenDA
+
+Execute the following command to start the docker containers:
+
+```
+docker compose up -d
+```
+
+The command will start the node and nginx containers and if you do `docker ps` you should see an output indicating all containers have status of “Up” with ports assigned.
+
+You may view the container logs using:
+
+```
+docker logs -f <container_id>
+```
+
+If you have successfully opted in to EigenDA and correctly running your EigenDA software, you should see the following logs for your EigenDA container:
+
+![EigenDA Logs](/img/operator-guides/eigenda-logs.png)
+
+The following example log messages confirm that your EigenDA node software is up and running:
```

Comment:
The logs look different now. Maybe update the logs from the latest version?

Reply:
+1